src/components/Preprocessor.js

```diff
@@ -149,6 +149,7 @@ class Preprocessor {
 
         src = new File(src);
 
+        output = File.stripPublicDir(output);
```

Comment:
I'mma be honest this feels super icky but seemed to be the only way for me to solve the problem. The File / path handling code seems to be somewhat messy. I wasn't sure how to work this into the general output path normalization. I'd be up for a larger refactoring of the file / path handling if you think it's warranted.

Reply:
I agree, the file path handling in general is rough across the entire codebase. Let's address it later in a separate commit after we have webpack 5 support ready to go.

Reply:
👍